bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/editor/ObservationEditor.scala

```diff
@@ -29,59 +31,115 @@ class ObservationEditor private (obs:Observation, canEdit:Boolean) extends StdMo
 
   // Editor component
   object Editor extends GridBagPanel with Rows {
-    addRow(new Label("Conditions"), new OptionLabel(obs.condition, ICON_CONDS))
-    addRow(new Label("Resources"), new OptionLabel(obs.blueprint, ICON_DEVICE))
+    private class RightLabel(t: String) extends Label(t) {
+      horizontalAlignment = Alignment.Right
+    }
+
+    private type Named = {
+      def name: String
+    }
+
+    private class OptionLabel(a:Option[Named], icon0:Icon) extends Label() {
+      horizontalAlignment = Alignment.Left
+      icon = icon0
+      text = a.map(_.name).getOrElse("none")
+      if (a.isEmpty)
+        foreground = Color.GRAY
```

Comment:
We may have used some Pinkish color for other labels when they are empty, can you check?